### DISCUSSION TITLE
Yarn install question，Why is this happening？

### DISCUSSION THREAD

**Discussion by 322326** [warengonzaga/css-text-portrait-builder#56] [Q&A] [answered]
![PixPin_2023-12-15_15-24-14](https://github.com/warengonzaga/css-text-portrait-builder/assets/101379104/30b79c7e-bbe8-457c-b24a-bf5dacfdf790)

**Comment by warengonzaga** [comment, marked as answer]
Hello @322326, thank you for reaching out. Unfortunately, I cannot replicate your issue.
If you can give me the step by step replication I might be able to help you.
